llvm/unittests/IR/MetadataTest.cpp

```diff
@@ -2177,7 +2177,7 @@ TEST_F(DICompileUnitTest, get) {
   MDTuple *Macros = getTuple();
   StringRef SysRoot = "/";
   StringRef SDK = "MacOSX.sdk";
-  bool CasFriendly = false;
+  bool CasFriendly = true;
```

Comment:
This change does not look like it belongs into this commit.

Reply:
I am not sure what you mean, there was a comment above where you asked me to make it true rather than false, or are you saying I shouldn't make a change to MetadataTest.cpp at all, I have to do something, because if I do not then clang will not build. getDisctinct requires a parameter for casfriendliness

Reply:
There are two commits in the PR, one for clang, one for LLVM, and this change seemed to be in the wrong one. Getting these details right will make upstreaming the patch much easier later.